startup_scripts/015_object_permissions.py

```diff
@@ -0,0 +1,44 @@
+import sys
+
+from users.models import ObjectPermission, AdminGroup, AdminUser
+from startup_script_utils import load_yaml
+from django.contrib.contenttypes.models import ContentType
+
+object_permissions = load_yaml("/opt/netbox/initializers/object_permissions.yml")
+
+if object_permissions is None:
+    sys.exit()
+
+
+for params in object_permissions:
+
+    object_permission, created = ObjectPermission.objects.get_or_create(
+        name=params['name'],
+        description=params['description'],
+        enabled=params['enabled'],
+        actions=params['actions']
+    )
+
+# Need to try to pass a list of model_name and app_label for more than just the current all objects.
+    #object_types = ContentType.objects.filter(app_label__in=params.pop("object_types"))
+    #object_permission.object_types.set(ContentType.objects.filter(app_label__in=params.pop("object_types")))
+    object_permission.object_types.set(ContentType.objects.all())
+    object_permission.save()
```

Comment:
I would love some suggestions on how to tackle this for individual object types.  I was thinking using object_types key as a list in the `initializers/object_permissions.yml`.  I was having a challenge with properly filtering the list of object_types against the nbshell (django orm) given the list of object_types.